include/alpaka/rand/RandStl.hpp

```diff
@@ -68,6 +68,35 @@ namespace alpaka
                 public:
                     std::mt19937 m_State;
                 };
+
+                //#############################################################################
+                //! The STL random device based on the local entropy pool.
```

Comment:
In view of #577 , maybe not call it STL in the new code?

Reply:
uff yes, copy paste from above xD